src/testUtils/truncateAllTables.js

```diff
@@ -0,0 +1,8 @@
+export default async ({ getDb }) => {
+  const db = await getDb();
+  await db.any("DELETE from scheduled_calls_table");
+  await db.any("DELETE from wards");
```

Comment:
I tried deleting from wards before, if a ward has a visit counted against it I think this will fail because of foreign keys - just something to bare in mind

Reply:
Calls are deleted first to avoid the constraint errors (and wards are deleted before hospitals etc)

Reply:
My issue was I had calls counted in `ward_visit_totals` against wards so couldn't delete a ward until I'd deleted the row from the totals table

Reply:
Ah I didn't see the constraint on ward_visit_totals. That needs to go first. Or we just delete while ignoring constraints